unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/Voxels/VoxelController.cs

```diff
@@ -112,11 +112,11 @@ public void SetEditObjectLikeVoxel()
 
     public void EndMultiVoxelSelection()
     {
-        List<DCLBuilderInWorldEntity> voxelEntities = biwEntityHandler.GetAllVoxelsEntities();
+        List<BIWEntity> voxelEntities = biwEntityHandler.GetAllVoxelsEntities();
 
-        foreach (DCLBuilderInWorldEntity voxelEntity in voxelEntities)
+        foreach (BIWEntity voxelEntity in voxelEntities)
         {
-            if (BuilderInWorldUtils.IsWithInSelectionBounds(voxelEntity.gameObject.transform, lastMousePosition, Input.mousePosition))
+            if (BIWUtils.IsWithInSelectionBounds(voxelEntity.rootEntity.gameObject.transform, lastMousePosition, Input.mousePosition))
```

Comment:
```suggestion
            if (BIWUtils.IsWithinSelectionBounds(voxelEntity.rootEntity.gameObject.transform, lastMousePosition, Input.mousePosition))
```

"Within" is a single word. So the name of the method should be `Within`, and not `WithIn`